roles/ceph-common/tasks/main.yml

```diff
@@ -64,24 +64,16 @@
   tags:
     - package-install
 
-- name: get ceph rhcs version
-  shell: rpm -q --qf "%{version}\n" ceph-common | cut -f1,2 -d '.'
+- name: get ceph version
+  shell: ceph --version | cut -f3 -d ' '
```

Comment:
Can we add something like `| tee out.txt ; test ${PIPESTATUS[0]} -eq 0` to that command so we can trap an eventual installation problem?
The issue here is the `| cut` always returning 0 even if the `ceph` command failed.